src/src.pro

```diff
@@ -104,8 +104,7 @@ unix:!macx {
         -lyajl \
         -lGLU \
         -lzip \
-        -lz \
-        -lopengl32
+        -lz
```

Comment:
Actually this only targets Linux platforms as the selector is `unix:!macx` i.e. all unixes _except_ macx (not sure if that is or is not the free "Darwin" or the pay (through the nose for) MacOs!